frontend/src/components/common/Filter/common/entities.ts

```diff
@@ -134,6 +136,7 @@ export enum CATEGORY_FILTER_ID {
  */
 export interface BaseCategoryFilterConfig {
   analyticsEvent?: EVENTS;
+  analyticsPayloadKey?: ANALYTICS_PAYLOAD_KEY;
```

Comment:
Added `analyticsPayloadKey` to handle updated analytics event format. For example, older filter events are tracked with `{"payload": ${value}}`, new events are tracked with `{${payloadKey}: ${value}}`.